### PR TITLE
Louisiana 2025 income tax updates

### LINKED ISSUE #5362: Louisiana 2025 income tax rate
[Bill](https://legis.la.gov/legis/ViewDocument.aspx?d=1391418) was passed to incorporate a 3% flat tax

### LINKED ISSUE #5363: Louisiana 2025 retirement income exemption
A [bill](https://legis.la.gov/legis/ViewDocument.aspx?d=1391418) was passed on increase the exemption amount from $6,000 to $12,000 which will be adjusted annual with inflation

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #5363
Fixes #5362

### CODE REVIEW

File: policyengine_us/parameters/gov/states/la/tax/income/exempt_income/retirement/cap.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ brackets:
       2001-01-01: 65
     amount:
       2001-01-01: 6_000
+      2025-01-01: 12_000
```
Comment (DTrim99) [inline policyengine_us/parameters/gov/states/la/tax/income/exempt_income/retirement/cap.yaml:12]: LA retirement and disability exemption

Wouldn't this affect the disability cap too?

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/la/tax/income/exempt_income/retirement/cap.yaml:12]: Yep, adjusted

**Review by DTrim99 — APPROVED** [review]
All of the parameters look good, and I really like the idea of using a true/false switch for states that convert to (or near) a flat tax. We should do the same for Iowa.

**Comment by PavelMakarchuk** [comment]
> All of the parameters look good, and I really like the idea of using a true/false switch for states that convert to (or near) a flat tax. We should do the same for Iowa.

We have the same structure in Iowa
[Switch](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/states/ia/tax/income/rates/by_filing_status/active.yaml)

**Comment by PavelMakarchuk** [comment]
> > All of the parameters look good, and I really like the idea of using a true/false switch for states that convert to (or near) a flat tax. We should do the same for Iowa.
> 
> We have the same structure in Iowa [Switch](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/states/ia/tax/income/rates/by_filing_status/active.yaml)

There are other states where we could refactor the code to add a switch to a flat tax

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**